CHANGES.md

```diff
@@ -30,6 +30,12 @@ OpenSSL 3.4
 
 ### Changes between 3.4 and 3.5 [xx XXX xxxx]
 
+ * Changed DTLS handling of invalid records.
+   Records with invalid MAC or invalid length used to generate fatal alert,
+   and are now silently dropped.
+
+   *Omer Kattan*
+
```

Comment:
As we've decided this is a bug fix eligible for backports to stable branches, this needs to go into a different section in CHANGES.md. Please rebase this to fresh master branch and put the note into this part of CHANGES.md: https://github.com/openssl/openssl/blob/1d160dbf39fbdba89389ddff54e45bacf278b04a/CHANGES.md?plain=1#L234